src/core_plugins/testbed/public/testbed.html

```diff
@@ -5,6 +5,10 @@
     you need to demonstate your functionality. Nothing need be preserved. -->
     <!-- CONTENT START -->
 
+    <div class="kuiControlledTable">
+      <loading-results results-name="all the things"></loading-results>
```

Comment:
Good point about how the current design doesn't play well with internationalization. I like the approach you are proposing but it then begs the question: should `loading-results` and `no-items` really be a single component, something like `items-message` or something else (naming is hard!)?